docs/source/troubleshooting/index.rst

```diff
@@ -183,3 +183,68 @@ compare-and-set operation here.
          True
 
 Clients should be able to take the backup lock again after this step.
+
+Migrating from conservatively swept tables to thoroughly swept tables
+=====================================================================
+
+Background and Motivation
+-------------------------
+
+Long running read transactions need to be able to distinguish between cells that were never written to, and cells that
+have been overwritten and are now being cleaned by sweep. A solution to this is to leave a sentinel that will always be
+read no matter what timestamp the transaction is running at when deleting a cell along with the normal Atlas-level
+delete (empty byte-array). This is how tables running with conservative sweep work. The downside is that for data that
+is transient - i.e. gets written to and deleted fairly quickly - this metadata stays behind forever, and even if the
+table is empty from an Atlas level, it will still have all of this metadata in the underlying key value service,
+resulting in unbounded disk usage growth the more cells are written to and deleted from. It also means that more data
+is potentially loaded from the underlying KVS when doing range scans making queries worse as time goes on.
```

Comment:
`is potentially loaded from the underlying KVS when doing range scans, making queries worse as time goes on.` (note the comma)